propulsion-cosmos-reactor/.template.confg/template.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I note there is a typo in the folder name: `confg` - that does not seem to be affecting the operation, but should be fixed

Reply:
Yup, totally will fix that